### PR TITLE
`dollar_format()` rely on `number_format()`

### PR DESCRIPTION
**Pull request by larmarange**
This fixes #79

### CODE REVIEW

**Comment by larmarange** [comment]
OK, I will wait for #146 to be merged, as it will produce additional merge conflicts

**Comment by larmarange** [comment]
news bullet added and merge conflict solved

File: R/formatter.r [hunk 1]
```diff
@@ -249,19 +249,27 @@ unit_format <- function(accuracy = 1, scale = 1, prefix = "",
 #' Currency formatter: round to nearest cent and display dollar sign.
 #'
 #' The returned function will format a vector of values as currency.
-#' Values are rounded to the nearest cent, and cents are displayed if
-#' any of the values has a non-zero cents and the largest value is less
-#' than `largest_with_cents` which by default is 100000.
+#' If `accuracy` is not specified, values are rounded to the nearest cent,
+#' and cents are displayed if any of the values has a non-zero cents and
+#' the largest value is less #' than `largest_with_cents` which by default
```
Comment (hadley) [inline R/formatter.r]: Missing newline before `#`?

Comment (larmarange) [reply R/formatter.r]: corrected

**Comment by larmarange** [comment]
merge conflict solved

**Pull request merged**